### DISCUSSION TITLE
FaceSwap not properly functioning when using refiner SD 1.5

### DISCUSSION THREAD

**Discussion by joacog86** [lllyasviel/Fooocus#2776] [General]
I'm using a refiner with a model in sd1.5 (sorry for the explanation since i dont quite understand fully all of this). But for what I see, it starts generating the image with the FaceSwap and using the refiner, and then when it goes to step 25 of generating it, it jumps to the base model and completely forgets about the FaceSwap.

<img width="662" alt="Captura de Pantalla 2024-04-18 a la(s) 23 24 00" src="https://github.com/lllyasviel/Fooocus/assets/84082508/22a44e21-faff-44af-8539-a9864e2339e3">

**Comment by xhoxye** [comment]
Yes, there is only i2i after the refiner switch, and it will lose all other control
